bibliopixel/drivers/visualizer.py

```diff
@@ -38,14 +33,15 @@ def __init__(self, num=0, width=0, height=0, pixelSize=15, port=1618, stayTop=Fa
             if "windows" in operating_system:
                 exe_string = "start python"
             elif "darwin" in operating_system:
-                exe_string = "python"
+                exe_string = "python2.7"
                 suffix = "&"
             else:
-                exe_string = "python"
+                exe_string = "python2.7"
```

Comment:
You might be right on that.  we can tweak it to make it work....

To be frank :-D this whole idea is horrible.  You should be using multiprocessing probably or something like that: https://docs.python.org/3.4/library/multiprocessing.html (since 2.6).